entrypoint.sh

```diff
@@ -22,12 +22,10 @@ KERNEL_SRC_URL="https://cdn.kernel.org/pub/linux/kernel/v4.x"
 KERNEL_SRC_ARCHIVE="linux-$(uname -r | cut -d- -f1).tar.xz"
 KERNEL_SRC_DIR="${KERNEL_SRC_DIR:-/build/usr/src/linux}"
 ROOT_OS_RELEASE="${ROOT_OS_RELEASE:-/root/etc/os-release}"
-NVIDIA_DRIVER_VERSION="${NVIDIA_DRIVER_VERSION:-390.116}"
-NVIDIA_DRIVER_DOWNLOAD_URL_DEFAULT="https://us.download.nvidia.com/XFree86/Linux-x86_64/${NVIDIA_DRIVER_VERSION}/NVIDIA-Linux-x86_64-${NVIDIA_DRIVER_VERSION}.run"
-NVIDIA_DRIVER_DOWNLOAD_URL="${NVIDIA_DRIVER_DOWNLOAD_URL:-$NVIDIA_DRIVER_DOWNLOAD_URL_DEFAULT}"
+NVIDIA_DRIVER_VERSION="${NVIDIA_DRIVER_VERSION}"
 NVIDIA_INSTALL_DIR_HOST="${NVIDIA_INSTALL_DIR_HOST:-/opt/nvidia}"
 NVIDIA_INSTALL_DIR_CONTAINER="${NVIDIA_INSTALL_DIR_CONTAINER:-/usr/local/nvidia}"
-NVIDIA_INSTALLER_RUNFILE="$(basename "${NVIDIA_DRIVER_DOWNLOAD_URL}")"
+NVIDIA_INSTALLER_RUNFILE="NVIDIA-Linux-x86_64-${NVIDIA_DRIVER_VERSION}.run"
```

Comment:
Where does the entrypoint.sh ${NVIDIA_DRIVER_VERSION}
As I understand it the version is hard coded in the Dockerfile line 32.
What happens if you do not provide the NVIDIA_DRIVER_VERSION="390.116" at runtime?
I'm probably missing something but I don't know what.

Reply:
Yes it is set on Dockerfile. The default value is in the Dockerfile. The build is done from our CI (CircleCI). The value contains the latest value tested.

[Here](https://github.com/giantswarm/kubernetes-gpu/pull/11/files#diff-1a523bd9fa0dbf998008b37579210e12R149) I have added a section to explain how a new driver is included.

So in case, you need to test a new version you follow these steps. And there is a [script](https://github.com/giantswarm/kubernetes-gpu/pull/11/files#diff-d1add0d17cc36ceaa0815b4091a4d20aR1) included to help to modify that version